### PR TITLE
IBX-816 - Revise Image and RichText extensibility doc

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | IBX-816  |
| Versions      | 3.3, master |

### CODE REVIEW

File: docs/extending/online_editor_plugin.md [hunk 1]
```diff
@@ -1,127 +1,59 @@
...
+!!! note
 
-## Creating the plugin
+    Online Editor is based on AlloyEditor, which in turn uses CKEditor.
+    As a result, you create custom Online Editor plugins in a way similar to 
+    [the way you do it in CKEditor](https://ckeditor.com/docs/ckeditor4/latest/guide/plugin_sdk_sample.html).
 
-First, create the plugin file in `assets/js/alloyeditor/plugins/date.js`:
+First, create the plugin file in `assets/js/online_editor/plugins/date.js`.
+The following code implements an `InsertDate` command and attaches it to the editor:
```
Comment (DominikaK) [inline docs/extending/online_editor_plugin.md:16]:
```suggestion
The following code implements an `InsertDate` method and attaches it to the editor:
```
I think?

File: docs/extending/richtext_block.md [hunk 2]
```diff
@@ -1,108 +1,50 @@
-# Creating custom RichText blocks [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+# Create custom RichText block [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
 
-To create a RichText [custom Page block](../guide/page/create_custom_page_block.md), you need to define its layout, provide templates, add a subscriber and register it as a service.
+In addition to existing blocks that you can use when creating a Page, you can 
+also create a custom blocks.
```
Comment (DominikaK) [inline docs/extending/richtext_block.md:4]:
```suggestion
also create custom blocks. 
```

File: docs/extending/richtext_block.md [hunk 3]
```diff
@@ -1,108 +1,50 @@
-# Creating custom RichText blocks [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+# Create custom RichText block [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
 
-To create a RichText [custom Page block](../guide/page/create_custom_page_block.md), you need to define its layout, provide templates, add a subscriber and register it as a service.
+In addition to existing blocks that you can use when creating a Page, you can
```
Comment (DominikaK) [inline docs/extending/richtext_block.md:3]: Do you need to repeat the info from the main Page block page? Maybe instead link to that page and indicate that this is just one specific case of a block?

File: docs/extending/richtext_block.md [hunk 4]
```diff
@@ -1,108 +1,50 @@
...
 ```
+!!! note
 
-This configuration defines a new block, its view and configuration templates, and attribute type `richtext` (line 15).
-Remember to provide an icon for the block in the `assets/images/blocks/` folder.
+    Make sure that you provide an icon for the block in the `assets/images/blocks/` folder.
 
-Next, create a subscriber that converts a string of data into XML.
-Create a `src/Event/Subscriber/RichTextBlockSubscriber.php` file containing:
+Then, create a subscriber that converts a string of data into an XML code.
```
Comment (DominikaK) [inline docs/extending/richtext_block.md:22]:
```suggestion
Then, create a subscriber that converts a string of data into XML code.
```

**Review by DominikaK — CHANGES_REQUESTED** [review]

**Review by DominikaK — APPROVED** [review]

File: code_samples/back_office/online_editor/config/packages/custom_buttons.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+                        heading:
+                            buttons:
+                                ezanchor:
+                                    priority: 100
+                                ezembedinline:
+                                    visible: false
+                        ezadd:
+                            buttons:
+                                hr:
+                                    priority: 50
```
Comment (damianz5) [inline code_samples/back_office/online_editor/config/packages/custom_buttons.yaml:16]:
```suggestion
                    toolbar:
                        group1:
                            priority: 60
                            buttons:
                                ibexaMoveUp:
                                    priority: 30
                                ibexaMoveDown:
                                    priority: 20
                                    visible: false
                                heading:
                                    priority: 10
```

those changes needs to be added on 4.x

**Review by damianz5 — APPROVED** [review]

**Pull request merged**